datalad/resources/procedures/cfg_text2git.py

```diff
@@ -10,7 +10,7 @@
     check_installed=True,
     purpose='configuration')
 
-annex_largefiles = '(not(mimetype=text/*))'
+annex_largefiles = '(not((mimetype=text/*)or(mimetype=inode/x-empty)))'
```

Comment:
Why bother with mimetype if it could be directly based on the file size? We did see mimetype detection getting new ideas/bugs so why not to KISS?

Reply:
> Why bother with mimetype if it could be directly based on the file size?

I'm guessing just because it is what flowed out of the discussion in gh-3663.  I'm fine going with the size condition.

Reply:
This comment confused me into thinking that the size comparison was considered and I missed it while glancing over that discussion. I reviewed it again and found no explicit consideration. Did I miss it or this ambigouse comment wasn't intended to be informative? ;-)

Reply:
> Did I miss it or this ambigouse comment wasn't intended to be informative? ;-)

By saying it "flowed out of the discussion" I intended to express that we reached that patch in a way that was dependent on the course of the discussion.  Roughly "an empty file isn't caught by mimetype=text/*" -> "oh, file --mime-type reports inode/x-empty" -> "ok, we could add that condition".  IOW, nobody (and particularly me who suggested the patch) stepped back and considered doing it without mimetype.

In general, given that human communication is lossy, I'd appreciate if you'd be more charitable and---rather than suggest that I was trying to be uninformative---assume that the information wasn't conveyed because it was poorly expressed or misread or some combination of both.

Reply:
> In general, given that human communication is lossy, I'd appreciate if you'd be more charitable and---rather than suggest that I was trying to be uninformative---assume that the information wasn't conveyed because it was poorly expressed or misread or some combination of both.

agree 100%
yeap -- let's us all be more charitable! ;)